File: 백준/Bronze/2566. 최댓값/최댓값.py
arr = []
for i in range(9):
  input_arr = list(map(int, input().split()))
  arr.append(input_arr)

splited_arr = []

for item in arr:
  temp = max(item)
  splited_arr.append(temp)

result = max(splited_arr)
col = -1
row = -1
for i in range(9):
  for j in range(9):
    if arr[i][j] == result:
      col = i
      row = j
      break

print(result)
print(f'{col + 1} {row + 1}')

File: 백준/Bronze/2566. 최댓값/최댓값.c
#include <stdio.h>

int main(void) {  
   int a[9][9], b[9][2];

   int max_row, max, temp = -1, col, row;
   for(int i = 0; i < 9; i++){
      for(int j = 0; j < 9; j++){
         scanf("%d", &a[i][j]);
      }
   }

   for(int i = 0; i < 9; i++){
      max_row = 0;
      max = 0;
      for(int j = 0; j < 9; j++){
         if(a[i][j] > max){
            max_row = j;
            max = a[i][j];
         }
      }
      b[i][0] = max;
      b[i][1] = max_row;
   }

   for(int i = 0; i < 9; i++){
      if(b[i][0] > temp){
         temp = b[i][0];
         col = i;
         row = b[i][1];
      }
   }

   printf("%d\n", temp);
   printf("%d %d", col + 1, row + 1);
   
}